src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -87,8 +88,11 @@ UCS_PROFILE_FUNC(ucs_status_t, uct_cuda_copy_mem_reg,
         return UCS_OK;
     }
 
+    log_level = (flags & UCT_MD_MEM_FLAG_HIDE_ERRORS) ? UCS_LOG_LEVEL_DEBUG :
```

Comment:
maybe declare `log_level` with `const` qualifier and assign the value during initialization, wdyt?
otherwise - align by `=` with the statement below